<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatRecipeWizardDialogBase</class>
 <widget class="QDialog" name="PatRecipeWizardDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recipe Wizard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="gex.qrc">
    <normaloff>:/gex/icons/gex_application_48x48.png</normaloff>:/gex/icons/gex_application_48x48.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QLabel" name="TextLabelTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Ms Sans Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Easy&lt;i&gt;Recipe&lt;/i&gt;: Confirm tests to have under PAT...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer36">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>65</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Skip this Wizard page...and keep ALL tests in recipe</string>
       </property>
       <property name="text">
        <string>Back &lt;&lt;</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Skip this Wizard page...and keep ALL tests in recipe</string>
       </property>
       <property name="text">
        <string>Skip...</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="maximumSize">
        <size>
         <width>65</width>
         <height>26</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Finish recipe creation</string>
       </property>
       <property name="text">
        <string>Next &gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tests to DISABLE</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel2_2_2_2_2_2_4_2_2_5">
         <property name="maximumSize">
          <size>
           <width>32767</width>
           <height>32767</height>
          </size>
         </property>
         <property name="text">
          <string>Below are the tests Quantix suggests to &lt;b&gt;DISABLE&lt;/b&gt; from PAT processing. &lt;b&gt;Right-click&lt;/b&gt; any test you want to &lt;b&gt;keep&lt;/b&gt; them into the PAT recipe..</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="buttonTestsToEnable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>KEEP selected tests into PAT recipe</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="gex.qrc">
             <normaloff>:/gex/icons/recycle.png</normaloff>:/gex/icons/recycle.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer37">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>582</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="listViewDisabledTests">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Test#</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Test Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Low L.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>High L.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Wizard advice</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Why disabling this test...</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TabPage">
      <attribute name="title">
       <string>Tests PAT-Ready</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QLabel" name="textLabel2_2_2_2_2_2_4_2_2">
         <property name="maximumSize">
          <size>
           <width>32767</width>
           <height>32767</height>
          </size>
         </property>
         <property name="text">
          <string>Below are the tests Quantix suggests to &lt;b&gt;KEEP&lt;/b&gt; for PAT processing. &lt;b&gt;Right-click&lt;/b&gt; any test you want to &lt;b&gt;disable&lt;/b&gt; from PAT and exclude them from the PAT recipe..</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <widget class="QPushButton" name="buttonTestsToDisable">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>32</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>DISABLE selected tests from PAT</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="gex.qrc">
             <normaloff>:/gex/icons/file_remove.png</normaloff>:/gex/icons/file_remove.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer38">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>321</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="listViewEnabledTests">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Test#</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Test Type</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Low L.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>High L.</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Wizard advice </string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>comments</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="gex.qrc"/>
 </resources>
 <connections/>
</ui>
